modules/mapper-extras/src/main/java/org/elasticsearch/index/mapper/ScaledFloatFieldMapper.java

```diff
@@ -522,6 +525,12 @@ public SortField sortField(@Nullable Object missingValue, MultiValueMode sortMod
             return new SortField(getFieldName(), source, reverse);
         }
 
+        @Override
+        public BucketedSort newBucketedSort(BigArrays bigArrays, Object missingValue, MultiValueMode sortMode, Nested nested,
```

Comment:
I haven't written tests for plugging in these abstractions other than the tests for `top_metrics` itself. I'd love some advice on what is worth it. This one probably isn't, but `DoubleValuesComparatorSource` probably is. Same for the sort implementations, I guess. Thoughts?

Reply:
I'm not sure tbh.  I'm inclined to say that since top_metrics is the only consumer at the moment, the current testing is sufficient.